include/swift/AST/ParseRequests.h

```diff
@@ -192,6 +193,30 @@ class EvaluateIfConditionRequest
       bool shouldEvaluate) const;
 };
 
+/// Parse the '-define-availability' arguments.
+class AvailabilityMacroArgumentsRequest
+    : public SimpleRequest<AvailabilityMacroArgumentsRequest,
+                           AvailabilityMacroMap *(ASTContext *),
```

Comment:
I think this should hand back a `const` pointer

```suggestion
                           const AvailabilityMacroMap *(ASTContext *),
```